arcgis-ios-sdk-samples/Layers/Stretch renderer/StretchRendererViewController.swift

```diff
@@ -75,7 +75,7 @@ class StretchRendererViewController: UIViewController, StretchRendererSettingsVC
     
     //MARK: - Show/hide settings view
     
-    private func toggleSettingsView(on: Bool) {
-        self.visualEffectView.hidden = !on
+    fileprivate func toggleSettingsView(_ on: Bool) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage